app/views/ncr/work_orders/form.html.erb

```diff
@@ -114,6 +114,19 @@
           </ul>
         <%- end %>
       </div>
+    <% elsif @model_instance.approved? %>
+      <div class="form-group">
+        <%= f.label :cl_number, 'CL number' %>
+        <%= f.text_field :cl_number, class: 'form-control' %>
+      </div>
+      <div class="form-group">
+        <%= f.label :function_code %>
+        <%= f.text_field :function_code, class: 'form-control' %>
+      </div>
+      <div class="form-group">
+        <%= f.label :soc_code, 'SOC code' %>
```

Comment:
You'll want to set these labels in the translation file instead so that the caps are correct on the detail page, etc.